addons/message_whats_new_v2.13/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "api_version": "0.1",
+  "id": "message_whats_new_v2.13",
+  "name": "What's new message",
+  "type": "message",
+  "conditions": {
+    "min_client_version": "2.13.0",
+    "max_client_version": "2.13.9"
+  },
```

Comment:
if we are not 100% sure of the translations: `translatable: false`

Reply:
Let's wait for Chris's responses... If we are still undure after that I'll add the flag.